### PR TITLE
Move all displays into separate structure

### PR DESCRIPTION
**Pull request by anetteu**

### CODE REVIEW

**Review by mrica-equinor — APPROVED** [review]
LGTM :) just rebase it pls

File: frontend/src/components/Displays/MissionButtons/MissionControlButtons.tsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ import styled from 'styled-components'
 import { Typography } from '@equinor/eds-core-react'
 import { useLanguageContext } from 'components/Contexts/LanguageContext'
 import { useMissionControlContext } from 'components/Contexts/MissionControlContext'
-import { StopMissionDialog, MissionStatusRequest } from './StopDialogs'
+import { StopMissionDialog, MissionStatusRequest } from '../../Pages/FrontPage/MissionOverview/StopDialogs'
```
Comment (eivindsjovold) [inline frontend/src/components/Displays/MissionButtons/MissionControlButtons.tsx:9]: Do we use relative imports?

File: frontend/src/components/Pages/FrontPage/MissionOverview/MissionQueueView.tsx [hunk 2]
```diff
@@ -9,7 +9,7 @@ import { ScheduleMissionDialog } from './ScheduleMissionDialog'
 import { Robot } from 'models/Robot'
 import { useLanguageContext } from 'components/Contexts/LanguageContext'
 import { useInstallationContext } from 'components/Contexts/InstallationContext'
-import { CreateMissionButton } from './CreateMissionButton'
+import { CreateMissionButton } from '../../../Displays/MissionButtons/CreateMissionButton'
```
Comment (eivindsjovold) [inline frontend/src/components/Pages/FrontPage/MissionOverview/MissionQueueView.tsx:12]: Relative import?

File: frontend/src/components/Pages/FrontPage/MissionOverview/OngoingMissionCard.tsx [hunk 3]
```diff
@@ -3,13 +3,13 @@ import { config } from 'config'
 import { tokens } from '@equinor/eds-tokens'
 import { Mission } from 'models/Mission'
 import styled from 'styled-components'
-import { MissionProgressDisplay } from './MissionProgressDisplay'
-import { MissionStatusDisplayWithHeader } from './MissionStatusDisplay'
+import { MissionProgressDisplay } from '../../../Displays/MissionDisplays/MissionProgressDisplay'
```
Comment (eivindsjovold) [inline frontend/src/components/Pages/FrontPage/MissionOverview/OngoingMissionCard.tsx:6]: Relative import?

File: frontend/src/components/Pages/FrontPage/MissionOverview/OngoingMissionCard.tsx [hunk 4]
```diff
@@ -3,13 +3,13 @@ import { config } from 'config'
...
+import { MissionStatusDisplayWithHeader } from '../../../Displays/MissionDisplays/MissionStatusDisplay'
```
Comment (eivindsjovold) [inline frontend/src/components/Pages/FrontPage/MissionOverview/OngoingMissionCard.tsx:7]: Relative import?

File: frontend/src/components/Pages/FrontPage/MissionOverview/OngoingMissionCard.tsx [hunk 5]
```diff
@@ -3,13 +3,13 @@ import { config } from 'config'
...
 import { useNavigate } from 'react-router-dom'
-import { MissionControlButtons } from './MissionControlButtons'
-import BatteryStatusView from '../RobotCards/BatteryStatusView'
+import { MissionControlButtons } from '../../../Displays/MissionButtons/MissionControlButtons'
```
Comment (eivindsjovold) [inline frontend/src/components/Pages/FrontPage/MissionOverview/OngoingMissionCard.tsx:9]: Relative import?

File: frontend/src/components/Pages/FrontPage/MissionOverview/OngoingMissionCard.tsx [hunk 6]
```diff
@@ -3,13 +3,13 @@ import { config } from 'config'
...
+import BatteryStatusDisplay from '../../../Displays/RobotDisplays/BatteryStatusDisplay'
```
Comment (eivindsjovold) [inline frontend/src/components/Pages/FrontPage/MissionOverview/OngoingMissionCard.tsx:10]: Relative import?

File: frontend/src/components/Pages/FrontPage/MissionOverview/OngoingMissionCard.tsx [hunk 7]
```diff
@@ -3,13 +3,13 @@ import { config } from 'config'
...
 import { BatteryStatus } from 'models/Battery'
-import { MissionRobotDisplay } from './MissionRobotDisplay'
+import { MissionRobotDisplay } from '../../../Displays/MissionDisplays/MissionRobotDisplay'
```
Comment (eivindsjovold) [inline frontend/src/components/Pages/FrontPage/MissionOverview/OngoingMissionCard.tsx:12]: Relative import?

File: frontend/src/components/Pages/FrontPage/MissionOverview/ScheduleMissionDialog.tsx [hunk 8]
```diff
@@ -4,7 +4,7 @@ import { useLanguageContext } from 'components/Contexts/LanguageContext'
 import { Icons } from 'utils/icons'
 import { useRef, useState, useEffect } from 'react'
 import { useInstallationContext } from 'components/Contexts/InstallationContext'
-import { CreateMissionButton } from './CreateMissionButton'
+import { CreateMissionButton } from '../../../Displays/MissionButtons/CreateMissionButton'
```
Comment (eivindsjovold) [inline frontend/src/components/Pages/FrontPage/MissionOverview/ScheduleMissionDialog.tsx:7]: RElative imports?

File: frontend/src/components/Pages/FrontPage/RobotCards/RobotStatusCard.tsx [hunk 9]
```diff
@@ -1,14 +1,14 @@
 import { Card, Typography } from '@equinor/eds-core-react'
 import { Robot } from 'models/Robot'
 import { tokens } from '@equinor/eds-tokens'
-import { RobotStatusChip } from './RobotStatusChip'
-import BatteryStatusView from './BatteryStatusView'
+import { RobotStatusChip } from '../../../Displays/RobotDisplays/RobotStatusChip'
```
Comment (eivindsjovold) [inline frontend/src/components/Pages/FrontPage/RobotCards/RobotStatusCard.tsx:4]: Relative import?

File: frontend/src/components/Pages/FrontPage/RobotCards/RobotStatusCard.tsx [hunk 10]
```diff
@@ -1,14 +1,14 @@
...
+import BatteryStatusDisplay from '../../../Displays/RobotDisplays/BatteryStatusDisplay'
```
Comment (eivindsjovold) [inline frontend/src/components/Pages/FrontPage/RobotCards/RobotStatusCard.tsx:5]: Relative import?

File: frontend/src/components/Pages/FrontPage/RobotCards/RobotStatusCard.tsx [hunk 11]
```diff
@@ -1,14 +1,14 @@
...
 import styled from 'styled-components'
-import { RobotImage } from './RobotImage'
+import { RobotImage } from '../../../Displays/RobotDisplays/RobotImage'
```
Comment (eivindsjovold) [inline frontend/src/components/Pages/FrontPage/RobotCards/RobotStatusCard.tsx:7]: Relative import?

File: frontend/src/components/Pages/FrontPage/RobotCards/RobotStatusCard.tsx [hunk 12]
```diff
@@ -1,14 +1,14 @@
...
 import { useNavigate } from 'react-router-dom'
 import { BatteryStatus } from 'models/Battery'
 import { useLanguageContext } from 'components/Contexts/LanguageContext'
-import PressureStatusView from './PressureStatusView'
+import PressureStatusDisplay from '../../../Displays/RobotDisplays/PressureStatusDisplay'
```
Comment (eivindsjovold) [inline frontend/src/components/Pages/FrontPage/RobotCards/RobotStatusCard.tsx:11]: Relative import?

File: frontend/src/components/Pages/MissionHistoryPage/HistoricMissionCard.tsx [hunk 13]
```diff
@@ -1,7 +1,7 @@
 import { Button, Table, Typography } from '@equinor/eds-core-react'
 import { config } from 'config'
 import { Mission } from 'models/Mission'
-import { MissionStatusDisplay } from '../FrontPage/MissionOverview/MissionStatusDisplay'
+import { MissionStatusDisplay } from '../../Displays/MissionDisplays/MissionStatusDisplay'
```
Comment (eivindsjovold) [inline frontend/src/components/Pages/MissionHistoryPage/HistoricMissionCard.tsx:4]: Relative import?

File: frontend/src/components/Pages/RobotPage/RobotPage.tsx [hunk 14]
```diff
@@ -7,12 +7,12 @@ import styled from 'styled-components'
 import { BackButton } from '../../../utils/BackButton'
 import { LocalizationSection } from './LocalizationSection'
 import { Header } from 'components/Header/Header'
-import { RobotImage } from '../FrontPage/RobotCards/RobotImage'
+import { RobotImage } from '../../Displays/RobotDisplays/RobotImage'
```
Comment (eivindsjovold) [inline frontend/src/components/Pages/RobotPage/RobotPage.tsx:10]: Relative import?

File: frontend/src/components/Pages/RobotPage/RobotPage.tsx [hunk 15]
```diff
@@ -7,12 +7,12 @@ import styled from 'styled-components'
...
 import { MoveRobotArm } from './RobotArmMovement'
-import PressureStatusView from '../FrontPage/RobotCards/PressureStatusView'
-import BatteryStatusView from '../FrontPage/RobotCards/BatteryStatusView'
+import PressureStatusDisplay from '../../Displays/RobotDisplays/PressureStatusDisplay'
```
Comment (eivindsjovold) [inline frontend/src/components/Pages/RobotPage/RobotPage.tsx:12]: Relative import?

File: frontend/src/components/Pages/RobotPage/RobotPage.tsx [hunk 16]
```diff
@@ -7,12 +7,12 @@ import styled from 'styled-components'
...
+import BatteryStatusDisplay from '../../Displays/RobotDisplays/BatteryStatusDisplay'
```
Comment (eivindsjovold) [inline frontend/src/components/Pages/RobotPage/RobotPage.tsx:13]: Relative import?

File: frontend/src/components/Pages/RobotPage/RobotPage.tsx [hunk 17]
```diff
@@ -7,12 +7,12 @@ import styled from 'styled-components'
...
 import { BatteryStatus } from 'models/Battery'
-import { RobotStatusChip } from '../FrontPage/RobotCards/RobotStatusChip'
+import { RobotStatusChip } from '../../Displays/RobotDisplays/RobotStatusChip'
```
Comment (eivindsjovold) [inline frontend/src/components/Pages/RobotPage/RobotPage.tsx:15]: Relative import?

**Review by eivindsjovold — CHANGES_REQUESTED** [review]
Relative imports are not preferred in my opinion. Would have absolute imports from source.

File: frontend/src/components/Alerts/FailedMissionAlert.tsx [hunk 18]
```diff
@@ -2,7 +2,7 @@ import { Button } from '@equinor/eds-core-react'
 import { config } from 'config'
 import { Mission, MissionStatus } from 'models/Mission'
 import styled from 'styled-components'
-import { MissionStatusDisplay } from '../Pages/FrontPage/MissionOverview/MissionStatusDisplay'
+import { MissionStatusDisplay } from '../Displays/MissionDisplays/MissionStatusDisplay'
```
Comment (eivindsjovold) [inline frontend/src/components/Alerts/FailedMissionAlert.tsx:5]: Relative import?

File: frontend/src/components/Pages/MissionHistoryPage/HistoricMissionCard.tsx (hunk 13 above)
Comment (eivindsjovold) [inline frontend/src/components/Pages/MissionHistoryPage/HistoricMissionCard.tsx:4]: Relative import

**Review by eivindsjovold — CHANGES_REQUESTED** [review]
Still some relative imports

**Review by eivindsjovold — APPROVED** [review]
LGTM :)

**Pull request merged**